app/components/viral/distance_of_time_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Viral
+  # Card component for rendering sections of pages.
+  class DistanceOfTimeComponent < Viral::Component
+    attr_reader :time_difference, :original_time
```

Comment:
**suggestion (blocking):** Since this is a time ago component it probably does not need to take in the current time, looking through your code you are always passing in `Time.now`.  What I think you should do is set that as the default current_time and if someone wants to override it they can but at least they would not need to pass it in.

Reply:
Completed in ebf6676022ee1938aa3ba12f06ef58ba32d369ae